x-pack/plugins/apm/server/routes/transactions.ts

```diff
@@ -171,21 +171,27 @@ export const transactionLatencyChatsRoute = createRoute({
       searchAggregatedTransactions,
     };
 
-    const {
-      latencyTimeseries,
-      overallAvgDuration,
-    } = await getLatencyTimeseries({
-      ...options,
-      latencyAggregationType: latencyAggregationType as LatencyAggregationType,
-    });
+    const [latencyData, anomalyTimeseries] = await Promise.all([
+      getLatencyTimeseries({
+        ...options,
+        latencyAggregationType: latencyAggregationType as LatencyAggregationType,
+      }),
+      transactionName
+        ? undefined
+        : getAnomalySeries(options).catch((error) => {
+            logger.warn(`Unable to retrieve anomalies for latency charts.`);
+            logger.error(error);
+            return undefined;
+          }),
```

Comment:
nit: Since `transactionName` is part of `options` and already an argument in `getAnomalySeries` it might be cleaner to return early from within the function:

```suggestion
      getAnomalySeries(options).catch((error) => {
        logger.warn(`Unable to retrieve anomalies for latency charts.`);
        logger.error(error);
        return undefined;
      }),
```